apis/v1alpha1/shared_types.go

```diff
@@ -211,6 +211,19 @@ type RouteStatus struct {
 	//
 	// +kubebuilder:validation:MaxItems=100
 	Gateways []RouteGatewayStatus `json:"gateways"`
+
+	// AdmittedGateways is a list of Gateways that have admitted this route.
```

Comment:
Should we make sure that there is one way language we talk about this relationship? Is it "admitted" or "bound" (both terms exist in the repo).

Ask: can we decide on one and change all references to be consistent?

Reply:
I like bound more :)

Reply:
> Should we make sure that there is one way language we talk about this relationship?

+1, we currently use "Admitted" and should continue doing so IMO.